node/auth/authenticator.go

```diff
@@ -3,27 +3,26 @@ package auth
 import (
 	"context"
 	"fmt"
+	"time"
+
 	"github.com/Layr-Labs/eigenda/api"
 	grpc "github.com/Layr-Labs/eigenda/api/grpc/validator"
 	"github.com/Layr-Labs/eigenda/core"
 	gethcommon "github.com/ethereum/go-ethereum/common"
 	lru "github.com/hashicorp/golang-lru/v2"
-	"time"
 )
 
 // RequestAuthenticator authenticates requests to the DA node. This object is thread safe.
+// Returns the hash of the request and an error if the request is invalid.
 //
 // This class has largely been future-proofed for decentralized dispersers, with the exception of the
 // preloadCache method, which will need to be updated to handle decentralized dispersers.
 type RequestAuthenticator interface {
 	// AuthenticateStoreChunksRequest authenticates a StoreChunksRequest, returning an error if the request is invalid.
```

Comment:
doc wasn't updated to contain the meaning of the returned bytes

Reply:
Updated method doc and removed the description from the top level interface.